pkg/plugin/processor/builtin/registry.go

```diff
@@ -74,10 +81,10 @@ type ProcessorPluginConstructor func(log.CtxLogger) sdk.Processor
 func NewRegistry(
 	logger log.CtxLogger,
 	constructors map[string]ProcessorPluginConstructor,
-	schemaService pprocutils.SchemaService,
+	schemaRegistry schemaregistry.Registry,
```

Comment:
I don't like this change much, but the `avro.encode` and `avro.decode` processors need a schema registry, not a schema service.

Reply:
The thing that bothers me a bit is that the standalone and builtin registry now have each their own schema service. But I don't think it's a big issue right now, since the service is stateless, so I think we can live with this.